docs/datetime.txt

```diff
@@ -201,25 +201,24 @@ However, in some applications it is important that
 return different times, as they would normally if the application is
 run anywhere other than the UTC timezone.
 
-The best way to understand how to use
-:func:`~testfixtures.test_datetime` in these situations is to think of
-the internal queue as being a queue of :class:`~datetime.datetime`
-objects at the current local time with a `tzinfo` of None, much as
-would be returned by :meth:`~datetime.datetime.now`. 
-If you pass in a `tz` parameter to
+For these situations, you can explicitly specify what timezone the
+internal queue of :class:`~datetime.datetime` objects should be
+interpreted in by passing in a `tzinfo` to
+:func:`~testfixtures.test_datetime`. (When not specified, this
+defaults to UTC -- which is why both :meth:`~tdatetime.now` and
```

Comment:
Emphasis on ***interpreted***.

Yes the datetimes are naive, but there is still a concept of what timezone you're going to interpret them when I start talking to the test_datetime about timezones.

Reply:
Not really, and therein lies the rub: in "real use" there's localtime, when mocking we want to avoid that concept where possible, which is where the thing in the current docs comes from about "if you want utcnow and now to return different things"